cmd/do/do.go

```diff
@@ -321,6 +327,12 @@ func (verb *goVerb) Run(ctx context.Context, flags flag.FlagSet) error {
 	return nil
 }
 
+func (verb *jdocVerb) Run(ctx context.Context, flags flag.FlagSet) error {
```

Comment:
I haven't found anything good that would do it automatically. Probably because invoking javadoc is not as straight forward: it needs the .class files and the full classpath.

We could have a javadoc branch, where we check it in for each release, but I'm not sure it's worth it, because we're not really providing a library, where people will want to ready the javadoc.